doc/howto/index.rst

```diff
@@ -20,6 +20,7 @@ Getting started with autoinstall
    basic-server-installation
    configure-storage
    autoinstall-validation
+   Use ubuntu-image to build classical images <ubuntu-image-classic-examples>
```

Comment:
This may be a cache issue on a my side (even though I did a `make clean`) and I cannot really explain why but when testing locally the menu entry for the added page is visible on http://127.0.0.1:8000/howto/ (and on the page itself) but not on other how-to guides, ex. http://127.0.0.1:8000/howto/autoinstall-validation/. Do you see the same?

Reply:
That's a known Sphinx bug - though a `make clean-doc` should fix it. You can see it working in the RTD PR build, e.g. https://canonical-subiquity--2103.com.readthedocs.build/en/2103/tutorial/index.html